hip_data_tools/google/adwords.py

```diff
@@ -0,0 +1,115 @@
+"""
+This Module handles the connection and operations on Google AdWords accounts using adwords API
+"""
+from typing import List
+
+from attr import dataclass
+from googleads import oauth2, AdWordsClient
+from googleads.oauth2 import GoogleOAuth2Client
+
+from hip_data_tools.common import KeyValueSource, ENVIRONMENT, SecretsManager
+
+
+class GoogleAdWordsSecretsManager(SecretsManager):
+    """
+    Secrets manager for Google Adwords Specific secrets, these are modeled to handle the OAuth
+    based google client and refresh token secrets
+    Args:
+        source (KeyValueSource): The Key Value source which holds secrets
+        client_secret_var (str): The Key pointer for client_secret
+        refresh_token_var (str): The Key pointer for refresh_token
+        developer_token_var (str): The Key pointer for developer_token
+    """
+
+    def __init__(self,
+                 source: KeyValueSource = ENVIRONMENT,
+                 client_secret_var: str = "adwords_client_secret",
```

Comment:
should these be in all caps? Looks odd them being lower case

Reply:
Yes, Because it is a module level Constant